specs/FamilySafety.md

```diff
@@ -0,0 +1,64 @@
+Family Safety
+===
+
+# Background
+Provide end evelolper a new API to toggle Family Safety feature on and off. Once Family Safety is enabled, developer won't be able to turn it off while webview2 instance is running. Once enable, it will provide the same functionally as the browser like: Activity report, Safe Search and Web Filtering. Please see https://www.microsoft.com/en-us/microsoft-365/family-safety for details on Family Safety. 
+
+# Examples
+## WinRT and .NET   
+```c#
+void WebView_ToggleFamilySafetyFeature(object sender, object e)
+{
+    Control.CreationProperties.EnabledFamilySafety = true;
```

Comment:
What is `Control`? You don't show changes to the CreationProperties API in this document, do you intend to change that as well?

Instead for the .NET sample you could show creating the environmentoptions, setting your new property, and creating the environment.

Reply:
Circle back on since the dependency is getting unblocked. Updated the sample code